lib/packwerk/configuration.rb

```diff
@@ -79,7 +79,10 @@ def initialize(configs = {}, config_path: nil)
     end
 
     def load_paths
-      @load_paths ||= ApplicationLoadPaths.extract_relevant_paths(@root_path, "test")
+      Packwerk::Diagnostics.log('About to execute "bin/rake packwerk:dump_rails_dependencies_to_json"', __FILE__)
+      puts `bin/rake packwerk:dump_rails_dependencies_to_json`
+      Packwerk::Diagnostics.log('Finished executing command.', __FILE__)
+      raise 'Diagnostic tests complete'
```

Comment:
I raise to finish the test, but the plan here is to capture the results from shelling out, parse them as JSON, and incorporate them into the configuration. From there, we can use the simple Ruby primitives rather than the Rails stuff.